docs/DeveloperGuide.md

```diff
@@ -225,185 +236,193 @@ user in the `s-update` command
-![View a Single Ingredient Activity Diagram](images/IngredientViewSingleActivityDiagram.png) 
+![View a Single Ingredient Activity Diagram](images/IngredientViewSingleActivityDiagram.png)
 
 #### Design consideration:
-  
+
 ##### Aspect: How find the ingredient's level executes
-  
+
   * **Alternative 1 (current choice):** Obtain the ingredient name of the ingredient entered by the user, and use the
   ingredient name to find the ingredient by looping through the ingredient list.
     * Pros: Code is more readable.
-    * Cons: Every execution of the command will require one to access the name of the ingredient and loop through 
-    the list once, which may increase the time required for the operation. 
-    
-  * **Alternative 2:** Map the ingredient entered by the user to a index which corresponds to the index of the 
+    * Cons: Every execution of the command will require one to access the name of the ingredient and loop through
+    the list once, which may increase the time required for the operation.
+
+  * **Alternative 2:** Map the ingredient entered by the user to a index which corresponds to the index of the
   ingredient in the list, then find the ingredient using the index.
     * Pros: Do not require looping through the list every time `IngredientViewSingleCommand` executes.
-    * Cons: Code may be less readable.     
-  
+    * Cons: Code may be less readable.
+
 ### \[Completed\] Set ingredients' levels feature
 
-The completed set ingredients' levels feature consists of three commands with slightly different formats, which complement one another, to provide a set of useful commands for enhanced user experiences. The three commands are :
+The completed set ingredients' levels feature consists of three commands with slightly different command words and take in different numbers of parameters. The three commands complement one another, to provide a set of useful commands for enhanced user experiences in setting ingredients' levels. The three commands (including command word, prefix(es) if any and parameters taken in) are :
 
 * `i-set i/INGREDIENT_NAME m/AMOUNT` — Sets the level of one specific ingredient to the specified amount.
 * `i-set-default` — Sets the levels of all ingredients defined in the ingredient book to pre-determined amounts.
 * `i-set-all M/AMOUNT_FOR_MILK P/AMOUNT_FOR_PEARL B/AMOUNT_FOR_BOBA L/AMOUNT_FOR_BLACK_TEA G/AMOUNT_FOR_GREEN_TEA S/AMOUNT_FOR_BROWN_SUGAR` — Sets the levels of all ingredients defined in the ingredient book to different specified amounts for each ingredient.
 
+Please note that because tCheck is designed for an imaginary bubble tea branch, T-Sugar, which produces all its drinks using six ingredients,
+namely Milk, Pearl, Boba, Black Tea, Green Tea and Brown Sugar, all the six ingredients are pre-defined in tCheck's ingredient book. Only these six available ingredients' levels can be set using tCheck.
+
 #### Completed Implementation
```

Comment:
Remove "Completed" in the sub-header. This was a recommendation from the peer review we did.

Reply:
Noted, will delete , thanks.